src/helm/benchmark/run_specs/melt_run_specs.py

```diff
@@ -291,3 +294,109 @@ def get_math_spec(
+        "vi": "Vietnamese",
+        "en": "English",
+    }
+    source_language, target_language = language_pair.split("-")
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.melt_translation_scenario.MELTTranslationOPUS100Scenario",
+        args={"source_language": source_language, "target_language": target_language},
+    )
+
+    adapter_spec = get_machine_translation_adapter_spec(
+        source_language=FULL_LANGUAGE_NAMES[source_language],
+        target_language=FULL_LANGUAGE_NAMES[target_language],
+        max_train_instances=5 if fewshot else 0,
+    )
+
+    return RunSpec(
+        name=f"melt_translation_opus100:language_pair={language_pair},fewshot={fewshot}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_open_ended_generation_metric_specs(),
+        groups=["melt", "translation_opus100", f"translation_{language_pair}"],
+    )
+
+
+@run_spec_function("melt_translation_phomt")
+def get_melt_translation_phomt_spec(language_pair: str, fewshot: bool = False) -> RunSpec:
+    FULL_LANGUAGE_NAMES = {
+        "vi": "Vietnamese",
+        "en": "English",
+    }
+    source_language, target_language = language_pair.split("-")
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.melt_translation_scenario.MELTTranslationPhoMTScenario",
+        args={"source_language": source_language, "target_language": target_language},
+    )
+
+    adapter_spec = get_machine_translation_adapter_spec(
+        source_language=FULL_LANGUAGE_NAMES[source_language],
+        target_language=FULL_LANGUAGE_NAMES[target_language],
+        max_train_instances=5 if fewshot else 0,
+    )
+
+    return RunSpec(
+        name=f"melt_translation_phomt:language_pair={language_pair},fewshot={fewshot}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_open_ended_generation_metric_specs(),
+        groups=["melt", "translation_phomt", f"translation_{language_pair}"],
+    )
+
+
+@run_spec_function("melt_lm_mask_filling_mlqa")
+def get_melt_lm_mask_filling_mlqaa_spec(fewshot: bool = False) -> RunSpec:
+    scenario_spec = ScenarioSpec(class_name="helm.benchmark.scenarios.melt_lm_scenarios.MELTLMMaskFillingMLQAScenario")
+
+    instruction = "Hãy hoàn thành câu sau bằng cách điền vào các vị trí trống được đánh dấu bằng [MASK]."
+    adapter_spec = get_generation_adapter_spec(
+        instructions=instruction,
+        input_noun="Câu có chỗ trống",
+        output_noun="Câu đã hoàn thành",
+        max_train_instances=5 if fewshot else 0,
+        num_outputs=1,
+        max_tokens=1024,
+        temperature=0.0,
+    )
+
+    return RunSpec(
+        name=f"melt_lm_mask_filling_mlqa:fewshot={fewshot}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_language_modeling_metric_specs([]) + get_exact_match_metric_specs() + get_f1_metric_specs(),
```

Comment:
`get_language_modeling_metric_specs()` only works with `get_language_modeling_adapter_spec()` so you should remove it. Also, language modeling metrics only work with models that return logprobs, which means it won't work with models such as GPT-4o.